### PR TITLE
Trigger workflow only when PR has label 'test'

### PR DESCRIPTION
**Pull request by hershpa**
Signed-off-by: Hersh Pathak hersh.pathak@intel.com

### CODE REVIEW

**Comment by hershpa** [comment]
@vbedida79 please help review

**Comment by vbedida79** [comment]
lgtm

**Comment by hershpa** [comment]
@uMartinXu please help review.

Main change is to trigger workflow only when PR has label 'test'.
Trigger the workflow on pull_request event on main branch.
Some updates to comments for clarity.

File: .github/workflows/trigger-dci-apps.yml [hunk 1]
```diff
@@ -1,13 +1,18 @@
...
+name: Trigger DCI Apps Workflow
+on: 
+  workflow_dispatch:
+  pull_request:
+     types: [ labeled ]
+     branches: main
 jobs:
   using-trigger-and-wait:
     name: Remote Trigger for DCI Apps Job
+    if: ${{ github.event.label.name == 'test' }}
```
Comment (uMartinXu) [inline .github/workflows/trigger-dci-apps.yml:15]: Can we use a more readable label name other than "test"? :-)

File: .github/workflows/trigger-dci-apps.yml [hunk 2]
```diff
@@ -1,13 +1,18 @@
-# This is CI trigger for the Intel Redhat DCI-Apps Job
+# GH Action workflow to trigger CI facilitated by Intel Red Hat DCI Apps Job
```
Comment (uMartinXu) [inline .github/workflows/trigger-dci-apps.yml:1]: based on Intel Red Hat DCI apps job

File: .github/workflows/trigger-dci-apps.yml (hunk 1 above)
Comment (hershpa) [reply .github/workflows/trigger-dci-apps.yml:15]: Sure

Comment (hershpa) [reply .github/workflows/trigger-dci-apps.yml:15]: Any suggestions? Some ideas: code-test, test-code, ci-check, run-dci-app-agent etc

Comment (hershpa) [reply .github/workflows/trigger-dci-apps.yml:15]: @vbedida79 @uMartinXu

**Pull request closed without merging**